src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -593,8 +580,11 @@ private StudentEnrollDetails enrollStudent(StudentAttributes validStudentAttribu
 
     /* All empty lines or lines with only white spaces will be skipped.
      * The invalidity info returned are in HTML format.
+     *
+     * @param studentList is built to be used in enrollStudents(String, String, boolean)
      */
-    private List<String> getInvalidityInfoInEnrollLines(String lines, String courseId) throws EnrollException {
+    private List<String> getInvalidityInfoInEnrollLines(String lines, String courseId, List<StudentAttributes> studentList)
```

Comment:
If we were to throw exception for invalidInfo and return student list, all unit test would need to be changed for getInvalidityInfoInEnrollLines. Should I proceed to change all unit test?